example/lib/charts/candle_chart_screen.dart

```diff
@@ -107,30 +107,32 @@ class _CandleChartScreenState extends State<CandleChartScreen> {
                         .caption
                         .copyWith(fontSize: 13.0),
                   ),
-                  TargetAreaDecoration(
-                    targetMax: targetMax,
-                    targetMin: targetMin,
-                    colorOverTarget:
-                        Theme.of(context).colorScheme.error.withOpacity(1.0),
-                    targetLineColor: Theme.of(context).colorScheme.secondary,
-                    targetAreaFillColor: Theme.of(context)
-                        .colorScheme
-                        .secondary
-                        .withOpacity(0.3),
-                    lineWidth: 2.0,
-                    targetAreaRadius: BorderRadius.circular(12.0),
-                  ),
+                  // TargetAreaDecoration(
+                  //   targetMax: targetMax,
+                  //   targetMin: targetMin,
+                  //   colorOverTarget: Theme.of(context).colorScheme.error.withOpacity(1.0),
+                  //   targetLineColor: Theme.of(context).colorScheme.secondary,
+                  //   targetAreaFillColor: Theme.of(context).colorScheme.secondary.withOpacity(0.3),
+                  //   lineWidth: 2.0,
+                  //   targetAreaRadius: BorderRadius.circular(12.0),
+                  // ),
```

Comment:
Do we still need the commented out code?

Reply:
Removed